docs/reference/verifier.md

```diff
@@ -303,3 +303,4 @@ spec:
             - "*"
 
 ```
+Notes: In version v1.0.0-rc.7, Ratify updated the name of built-in verifier from `notaryv2` to `notation` in accordance with [`notaryproject spec`](https://notaryproject.dev/docs/faq/#notary-project-terms). As a result, `notaryv2` verifier is only supported in Ratify v1.0.0-rc.6 and earlier versions. If you want to upgrade to v1.0.0-rc.7 or later, please update the verifier name to `notation` in the configuration or CR. Additionally, please update `notaryv2` to `notation` in your constraint templates if verifier name was referenced. e.g. [example constraint template](../../library/notation-issuer-validation/template.yaml)
```

Comment:
Suggestion to make this into a linkable section to be included in the release note.

Reply:
Question, if customer have already have RC6 installed, if they don't modify any local CRDs, and try to install RC7 what error will they see?

Reply:
Good question! The actual behavior is a bit complex.
1. If users use Rego policy and enforce all verifiers should pass, then it will fail the verification. Below is the verification result:
![image](https://github.com/deislabs/ratify/assets/6919333/429303ff-7802-49c7-b2a1-8422c9719a0e)
We can see that `notaryv2` failed verification.
2. If users use Config policy, then it's a bit tricky. Executor will iterate through all verifiers to find the first one that can verify the image. Since we have both `notaryv2` and `notation` configured, it might use either one to verify as there is no determined order to iterate through the verifier map. Ratify will return true or false depending on which verifier it selects actually.

So we should recommend users to keep only one of them to avoid undetermined result.